Dockerfile.hub.staging

```diff
@@ -0,0 +1,38 @@
+FROM counterfactual/statechannels:0.5.13
+LABEL maintainer="Liam Horne <liam@l4v.io>" 
+
+WORKDIR /statechannels/hub
+
+# ASSUMPTIONS
+#
+# - That ./packages/hub has been built, so a lib directory exists
+# - That ./packages/nitro-protocol has been built, so the .json artifacts exist
+
+# TODO: Add package.json first, then do dependencies, then do rest of code
+
+COPY ./packages/hub /statechannels/hub
+COPY ./packages/nitro-protocol /statechannels/nitro-protocol
+
+# Remove the nitro-protocol dependency from @statechannels/hub
+RUN sed -ie "/@statechannels\/nitro-protocol/d" package.json
+
+# Remove the devtools devDependency from @statechannels/hub (avoid resolution)
+RUN sed -ie "/@statechannels\/devtools/d" package.json
```

Comment:
Mind explaining why this is needed?

Reply:
`yarn --production` still tries to resolve the `devDependencies` values, so it tries to find `@statechannels/devtools` on the `npm` registry, but it's not there.